tests/test_georaster.py

```diff
@@ -504,3 +504,26 @@ def test_value_at_coords(self):
         assert img.value_at_coords(x, y) == \
             img.value_at_coords(lon, lat, latlon=True) == \
             img.data[0, -1, -2]
+
+    def test_from_array(self):
+
+        # Test that from_array works if nothing is changed
+        # -> most tests already performed in test_copy, no need for more
+        img = gr.Raster(datasets.get_path('landsat_B4'))
+        out_img = gr.Raster.from_array(img.data, img.transform, img.crs, nodata=img.nodata)
+        assert np.array_equal(out_img.data, img.data)
```

Comment:
Could do `assert out_img == img` ;) But this works too!

Reply:
Good call!